### ISSUE TITLE
show all sub-statuses of a project 

### ISSUE THREAD

**Issue by haroon-sheikh** [sitture/commit-status#60]
## Task

* Currently when click on a project, it shows the commit history of the project. But we'd like to change that so that clicking on the project shows all of the sub-statuses of the project and each status links to details in a new tab/window.

E.g.

Project A
- Sub Status 1
- Sub Status 2

**Comment by jpacsai** [comment]
Hi :) can I work on this issue?

**Comment by haroon-sheikh** [comment]
@jpacsai Yes sure.. give it a go. Give me a shout if something is unclear.

**Comment by jpacsai** [comment]
hi, sorry but looks like I won't have enough time in the near future, so I'll pass :( hope somebody can pick this up an quickly fixes it
